### DISCUSSION TITLE
Interest in Extend interoperability

### DISCUSSION THREAD

**Discussion by octonawish-akcodes** [MDAnalysis/mdanalysis#4483] [GSoC Discussions]
Hi Team,

I am Abhishek Kumar, I am a well versed computer science student from India. I have been a past outreach intern as well where my work consist of adding a whole new feature to the project using Python and shell scripting. I believe the skills I gained there would be really helpful to this project idea. 

I am interested in Extend interoperability project idea under MDAnalysis, Can you provide me some base idea, any tracking issue or google document where the feature is being planned, any resources for this project idea to get me understand it better.

@hmacdope @yuxuanzhuang @xhgchen 

Thanks

**Comment by hmacdope** [comment]
Hi @octonawish-akcodes  check out the wiki with the project ideas listed for a better idea of what might be involved: https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-2024-Project-Ideas
